app/src/main/java/com/github/se/wanderpals/model/repository/TripsRepository.kt

```diff
@@ -522,6 +522,33 @@ class TripsRepository(
         }
       }
 
+  /**
+   * Checks if the specified trip ID exists in the 'Trips' collection. This method queries the
+   * Firestore database to verify the existence of a document corresponding to the given trip ID. It
+   * ensures that operations related to trip IDs are conducted with valid identifiers.
+   *
+   * @param tripId The unique identifier of the trip to be validated.
+   * @return Boolean indicating whether the trip ID is valid (true) or not (false). If the trip ID
+   *   exists in the database, returns true; otherwise, returns false. In the event of an exception
+   *   during the database query, the method also returns false,
+   */
+  private suspend fun isTripIdValid(tripId: String): Boolean =
```

Comment:
I think that making the function public won't expose anything significant and could be helpful in the future to reduce unnecessary PRs to open it up. It is, however, at your discretion